docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,2226 @@
+{
```

Comment:
<p>The <code>specific_logging</code> is similar in functionality to the <code>specific</code> parameter that can be passed via the <code>run_tardis()</code> function. It takes <strong>Boolean</strong> values, i.e., True or False. It is an optional parameter.</p><p> </p><p>Reworded to be similar structure as earlier sentence about logging_level. Added commas around i.e.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>